operate/webapp/src/main/java/io/camunda/operate/webapp/rest/dto/ListenerResponseDto.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.operate.webapp.rest.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class ListenerResponseDto {
+  private List<ListenerDto> listeners = new ArrayList<>();
+  private int totalCount;
```

Comment:
You can declare this as `long`, then you don't need to downcast the ES/OS response values. See for example `io.camunda.operate.webapp.rest.dto.listview.ListViewResponseDto` for comparison. I don't think this is a pratical problem (as it's unlikely there will be this many listeners), but the code becomes a bit cleaner.